packages/next/src/generators/next-auth/utils/constants.ts

```diff
@@ -1 +1 @@
-export const NEXT_AUTH_VERSION = '4.20.1';
+export const NEXT_AUTH_VERSION = 'beta';
```

Comment:
is that correct?

Reply:
Yeah we're using the latest beta version for now. Think v5 will be released officially very soon and it simplifies the code a good bit so made sense to make the early upgrade.